### ISSUE TITLE
[Feature request] Audio/Video Multistreams

### ISSUE THREAD

**Issue by trzyglow** [StefanLobbenmeier/youtube-dl-gui#69]
It would be practical to include a multistream download option for videos which have multiplexed video/audio streams.
it generally works in yt-dlp with the parameters:

--audio-multistreams
--video-multistreams
and
--no-audio-multistreams
--no-video-multistreams

for their respective states.

**Comment by StefanLobbenmeier** [comment]
Yeah this is something that does not work very well with the current design. I imagine in a better UI we can change the dropdown into a multi select and then pass the correct options to yt-dlp
